metadetect/lsst/measure.py

```diff
@@ -101,15 +101,39 @@ def detect_and_deblend(
         config=meas_config,
         schema=schema,
     )
+    # avoids a warning spamming for every object
+    afw_table.CoordKey.addErrorFields(schema)
 
     detection_config = SourceDetectionConfig()
+
+    # DM does not have config default stability.  Set all of them explicitly
```

Comment:
Is there a way for us to detect if new config fields have been added that we do not have defaults for? Maybe we can write a unit test?

Reply:
I don't know how to detect the config fields

Reply:
There must be a method somewhere. Did you look at the output of `dir` on the object? We could just write a test to compare that to some reference we record now.

Reply:
I found a `toDict` method

Reply:
I added a check for SourceDetectionConfig

Reply:
huzzh!